framework/src/main/java/org/checkerframework/framework/util/CheckerMain.java

```diff
@@ -131,7 +131,7 @@ protected void assertValidState() {
             assertFilesExist(Arrays.asList(javacJar, jdkJar, checkerJar, checkerQualJar));
         } else {
             // TODO: once the jdk11 jars exist, check for them.
-            assertFilesExist(Arrays.asList(javacJar, checkerJar, checkerQualJar));
```

Comment:
@smillst  Why are we not checking for `javacJar`?

Reply:
Because it's not required when using Java 9+.

Reply:
ok, Thanks for the explanation. I have to update my PR too once this is merged.